src/main/java/Event.java

```diff
@@ -0,0 +1,23 @@
+public class Event extends Task {
+    private String startTime;
+    private String endTime;
+
+    public Event (String description, String startTime, String endTime) {
+        super(description);
+        this.startTime = startTime;
+        this.endTime = endTime;
+    }
+
+    public String getStartTime() {
+        return startTime;
+    }
+
+    public String getEndTime() {
+        return endTime;
+    }
+
+    @Override
+    public String toString() {
+        return "[E]" + super.toString() + " " + "(from: " + startTime + " to:" + endTime + ")";
```

Comment:
I would recommend to have the label 'E' as a static constant outside the method.